tests/io/test_read_eea_aqerep_v2.py

```diff
@@ -42,14 +51,15 @@ def test_read(reader):
         assert isinstance(data, UngriddedData)
 
         for stat_idx, statid in enumerate(station_id[var_name]):
+            print(statid)
             try:
                 stat_data = data[statid]
                 # It makes no sense to test this for every station
                 if stat_idx == 1:
                     assert isinstance(stat_data, StationData)
 
                 assert stat_data[var_name].mean() == pytest.approx(
-                    station_means[var_name][stat_idx], 0.01
+                    station_means[var_name][stat_idx], TEST_RTOL
                 )
             except:
                 print(f"failed test var {var_name}")
```

Comment:
Why are you capturing the assert?
Without the assert this test will pass no matter what,
and the print will be never seen.

Reply:
leftover from testing; removed